lib/split/algorithms/weighted_sample.rb

```diff
@@ -1,39 +0,0 @@
-module Split
```

Comment:
since we don't store the alternative anymore, anything that's not deterministic doesn't work anymore... no point keeping it around if it's broken. more likely to lead to confusion/accidentally trying to use it.